missions/templates/missions/mission_detail.html

```diff
@@ -19,9 +19,9 @@ <h2>Assignments</h2>
     <tbody>
     {% for assignment in object.assignment_set.all %}
     <tr>
-        <td>{{ assignment.position.name }}</td>
-        <td>{{ assignment.user.first_name }} {{ assignment.user.last_name }}</td>
-        <td>{{ assignment.confirmed }}</td>
+        <td {% if assignment.user in multiple_assigned_users %}class="text-danger"{% endif %}>{{ assignment.position.name }}</td>
```

Comment:
I think I'd prefer an additional column which contains a subtle warning symbol. If someone chooses to create multiple assignments it might be annoying to (literally) see red from then on.

Reply:
Now i add an extra column in front of the others if theres an multiple assignment for the mission. In that column is an warning icon for every line with a person assigned more than one. If you click on the icon you get an javascript alert with further information. 

One thing which is not considered in this solution is, that someone could be assigned in another mission at the same time. This is an important feature if they are planning for more than one ship.
If will open an issue for that